catkin_tools/verbs/catkin_build/job.py

```diff
@@ -139,6 +141,25 @@ def __init__(self, package, package_path, context, force_cmake):
         Job.__init__(self, package, package_path, context, force_cmake)
         self.commands = self.get_commands()
 
+    def get_multiarch(self):
+        if not sys.platform.lower().startswith('linux'):
+            return ''
+        # this function returns the suffix for lib directories on supported systems or an empty string
+        # it uses two step approach to look for multiarch: first run gcc -print-multiarch and if
+        # failed try to run dpkg-architecture
+        p = subprocess.Popen(
```

Comment:
It would be good to either check that the executable exists before calling the subprocess or by catching these types of errors to be more graceful when there is an error.